content/millennium/r4/financial/charge-item.md

```diff
@@ -29,14 +29,15 @@ The following fields are returned if valued:
 * [Entered Date](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.enteredDate){:target="_blank"}
 * [Reason](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason){:target="_blank"}
 * [Account](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.account){:target="_blank"}
-* [Extensions including custom attribute, description, modifier, net price, offset by, performing location, procedure, replacing, revenue code, and unit price](#extensions){:target="_blank"}
+* [Extensions including bill code schedule, custom attribute, description, modifier, net price, offset by, performing location, procedure, replacing, revenue code, and unit price](#extensions){:target="_blank"}
```

Comment:
We don't have other 'sub-field' extensions listed here. We should make them consistent and list all of them here.
@alexbeckner1

Reply:
Are the other subfield extensions you are referring to from the modifier and procedure codes like priority and modifier-code and procedure-code?

Reply:
Discussed offline and determined the way it is now is fine.